### DISCUSSION TITLE
Unable to get/install plugins

### DISCUSSION THREAD

**Discussion by mase1981** [kimai/kimai#3837] [Questions & Answers] [answered]
Hey, i am new to kimai, i just installed it on Synology docker. 
Following instructions for a plugin (https://www.kimai.org/store/maximiliangross-easy-backup-bundle.html) i am following the steps but unable to have it visible (working via the UI). 

in synology docker i have these folders mounted and accesible: 
/opt/kimai/var
opt/kimai/public

i then went ahead and created these folders (as they were missing) 
/opt/kimai/var/plugins
/opt/kimai/var/cache

I put the folder EasyBackupBundle-1.4.2 in the plugins folder and restarted my container. 
When i go to the Kimai UI --> plugins it tell me "You have no pluggins installed". 
I went back to the newly created folder i created earlier (/opt/kimai/var/cache) and i now see 2 new folders: 
/opt/kimai/var/cache/prod
/opt/kimai/var/cache/pro_

which led me to believe that the cache folder structure work. 
i tried to :
rm -r /opt/kimai/var/cache/prod/*
rebooted the container and still not working 

in kimai UI docker sidebar all permissions and everything else is green
Does anyone can tell me what i am missing? TIA

**Comment by BeckeBauer** [comment, marked as answer]
Did you rename the Bundle to `EasyBackupBundle`? If yes, try to reload the cache while running the container:

docker --> Kimai container --> Details --> Terminal --> create --> bash --> cd /opt/kimai --> `bin/console kimai:reload --env=prod`
<img width="938" alt="Bildschirm­foto 2023-02-08 um 21 17 07" src="https://user-images.githubusercontent.com/51003649/217641844-ba52108d-ff88-44ed-9ea6-c510d02412c2.png">

**Reply by mase1981** [reply]
Thank you so much, i did not rename the folder. This (and the command) worked, thank you so much !
